TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/core/objects/Proxy.java

```diff
@@ -129,7 +129,6 @@ public void start() {
 
     @Override
     public void stop() {
-        unregister();
```

Comment:
Why don't we need that anymore? Will it unregister automatically?

Reply:
If you restart the proxy with "restart proxy", the proxy will be unregistered directly in CORE after the command. But during the stop process of the server all players are kicked and this triggers the PlayerDisconnectEvent. 
And then there is the ProxyObject == null;

If the server is completely stopped, it will be tried to unregister it again anyway.